### ISSUE TITLE
Output.Linkify Multi-select

### ISSUE THREAD

**Issue by perrylackowski** [pyrevitlabs/pyRevit#2107]
**Is your feature request related to a problem? Please describe.**
I write a lot of pyRevit scripts to flag errors in my project. For example, I may need to adjust the NEMA ratings of some electrical equipment based on their locations in the building. Right now, my script outputs a list of all the equipment that's using the incorrect ratings, and I can click on their links one at a time to update the Enclosure parameters. The downside is that I have to click on them _one at a time_, so the process can be time consuming if there are many errors to fix.

**Describe the solution you'd like**
I'd like Output.Linkify to have an option flag called 'append_select'. If set to True, the output link buttons would append the items in the link to the existing selection, rather than creating a new selection set.

**Describe alternatives you've considered**
I could rebuild my code so that it puts all the IDs with errors into a single linkify button. However this does not always work because I may need to exclude some of the items after review.

**Comment by jmcouffin** [comment]
Hi Perry,
Did you know you can feed the linkify tool with a list?
Therefore, it should not be too hard to build up your list and group them to be able to select them in batch.

**Comment by perrylackowski** [comment]
Hi @jmcouffin!

Yes, I considered that, but on top of the difficulty that comes from restructuring my code, there are situations where I wouldn't want to group the links. For example, in the screenshot below, I might want to grab all of the elements and set the NEMA rating to 'NEMA 12', but I happen to know that one of the rooms is currently mislabeled by the architect, so I'd want to exclude some of that equipment from my selection. I wouldn't be able to do that if I just had one link for the whole list.
![image](https://github.com/eirannejad/pyRevit/assets/102748073/8c22a7f6-b672-4e76-bc85-fb3a62a6390e)

**Comment by sanzoghenzo** [comment]
I was thinking about this, and I believe the most user-friendly way to do this is to handle a CTRL (or SHIFT) + click on the link.

We can add a little javascript to listen to mouse click events and detect the [ctrlKey](https://www.youtube.com/watch?v=rdNRrBX4cQE) property of the event argument to add the "&extendSelection=true" query string to the link;
after that we need to handle that query string in the `ScriptConsoleUtils.ProcessUrl` method.
It will basically add another parameter to the `SelectElements` method that will set the `elementIdsToIsolate` object to the current selection instead of an empty list of ElementIds.

Not that hard to do, but I don't know if I have much time to actually implement it.

**Comment by perrylackowski** [comment]
I agree; CTRL+click is a better approach. I wasn't sure what was possible between the console log and the Revit interface. Sadly I have no experience with Javascript yet or this is something I would look into tackling myself.
